spec/lib/pivotal-tracker/special_character_password_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
+
+describe PivotalTracker do
+  context "account with special characters in password" do
+    describe "GET me" do
+      before do
+        PivotalTracker::Client.token = ENV['PIVOTAL_TOKEN_ID']
+        VCR.insert_cassette 'profile', :record => :new_episodes
+      end
+
+      after do
+        VCR.eject_cassette
+      end
+
+      it "records the fixture" do
+        PivotalTracker.get('/me')
```

Comment:
@kheppenstall http://stackoverflow.com/questions/7627419/how-to-use-basic-authentication-with-httparty-in-a-rails-app

Maybe we look into overriding the password in the `.get()` arguments. This might require making a kind of test user, and adding those credentials to `.env`.

E.g.,
```
auth = {:username => "test", :password => "!@#$%^&*"}
PivotalTracker.get("/me", :basic_auth => auth)
```